app/src/main/java/pl/ludek/smat/home/pool_office_client/data/apiservice/PoolInfoClient.kt

```diff
@@ -0,0 +1,37 @@
+package pl.ludek.smat.home.pool_office_client.data.apiservice
+
+import pl.ludek.smat.home.pool_office_client.domain.model.PoolInfoData
+import pl.ludek.smat.home.pool_office_client.domain.model.RelayData
+import retrofit2.Call
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object PoolInfoClient {
+    private const val BASE_URL = "http://10.0.2.2:8080/"
+
+    private val retrofit = Retrofit.Builder()
+        .baseUrl(BASE_URL)
+        .addConverterFactory(GsonConverterFactory.create())
+        .build()
+
+    private val service: ApiPoolInfoService = retrofit.create(ApiPoolInfoService::class.java)
+
+    fun switchFirst(onOff: Boolean): Call<RelayData> {
+        val state = if(onOff) 1 else 0
+        return service.switchRelay(0, state)
+    }
+
+    fun switchFive(onOff: Boolean): Call<RelayData> {
+        val state = if(onOff) 1 else 0
+        return service.switchRelay(5, state)
+    }
```

Comment:
Please be more generic, there is no need for concrete methods, let the method caller decide which relay he is working with:
```
fun switchRelay(relayId: Int, onOff: Boolean): Call<RelayData> {
    val state = if(onOff) 1 else 0
    return service.switchRelay(relayId, state)
}
```